src/viam/ur/module/ur_arm.cpp

```diff
@@ -389,15 +391,12 @@ void URArm::configure_(const std::unique_lock<std::shared_mutex>& lock, const De
     });
 
     // extract relevant attributes from config
-    current_state_->host = find_config_attribute<std::string>(cfg, "host");
-    current_state_->speed.store(degrees_to_radians(find_config_attribute<double>(cfg, "speed_degs_per_sec")));
-    current_state_->acceleration.store(degrees_to_radians(find_config_attribute<double>(cfg, "acceleration_degs_per_sec2")));
-    try {
-        auto threshold = find_config_attribute<double>(cfg, "reject_move_request_threshold_deg");
-        current_state_->reject_move_request_threshold_rad.emplace(degrees_to_radians(threshold));
-    }  // will go away once find_config_attribute returns an optional
-    catch (...) {
-        (void)0;
+    current_state_->host = find_config_attribute<std::string>(cfg, "host").value();
+    current_state_->speed.store(degrees_to_radians(find_config_attribute<double>(cfg, "speed_degs_per_sec").value()));
+    current_state_->acceleration.store(degrees_to_radians(find_config_attribute<double>(cfg, "acceleration_degs_per_sec2").value()));
+    auto threshold = find_config_attribute<double>(cfg, "reject_move_request_threshold_deg");
+    if (threshold) {
+        current_state_->reject_move_request_threshold_rad.emplace(degrees_to_radians(*threshold));
```

Comment:
Or partially specialize `degrees_to_radians` for `std::optional<T>`. Just kidding, don't do that.